ci/envs/37-minimal.yaml

```diff
@@ -19,7 +19,7 @@ dependencies:
   - rtree
   - matplotlib
   - matplotlib=3.1
-  - mapclassify>=2.2.0
+  # - mapclassify>=2.4.0  # this currently doesn't build due to conflicts
```

Comment:
Is this still the case?

Reply:
Let's check.

Reply:
Yup, still the case. I've commented it out again, we can check after the next update of minimal requirements.

Reply:
Trying it locally to look into the conflict error, I notice the following:

```
Package pandas conflicts for:
mapclassify=2.4.0 -> pandas[version='>=1.0']
mapclassify=2.4.0 -> networkx -> pandas[version='>=1.1']
pandas=0.25
```

So if you are using an older pandas (which we officially still support), that's basically impossible if using conda. So for those users, this essentially also bumps the pandas version .. 
(which I don't think I would mind, but maybe we should do that explicitly? Or can leave that for later)

Reply:
I am not sure what you're suggesting to do explicitly. Expand the error message saying that you need mapclassify 2.4.0 _and_ pandas 1.1? 

We support pandas 0.25 but for basic runtime. There are cases where we raise an error. Plotting with mapclassify is optional so I don't think there's an issue that it requires newer pandas (which is still more than a year old).

Reply:
I meant explicitly making pandas 1.0 a minimum requirement for GeoPandas proper. But you're right, this is only an optional plotting feature, and we still support older pandas for basic features. 

(we should still consider bumping the pandas minimum version, though, but that's for a separate issue :))